videoplayer/static/js/videoplayer_view.js

```diff
@@ -4,16 +4,63 @@ function videoPlayerXBlockInitView(runtime, element) {
      * In the LMS, element is the DOM container.
      * In the CMS, element is the jQuery object associated*
      * So here I make sure element is the jQuery object */
-    // if(element.innerHTML) element = $(element);
-      
-    // var video = element.find('video:first');
+    if(element.innerHTML) element = $(element);
+
+    var video = element.find('video:first');
```

Comment:
we should get all videos not just first one and loop them and initialize all the videos.